### PR TITLE
feat: enable Swahili language

### PR DESCRIPTION
**Pull request by larymak**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by sidemt — APPROVED** [review]
LGTM 👍

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
image

We should get the Client UI properly translated before we launch.

**Comment by larymak** [comment]
> image
> We should get the Client UI properly translated before we launch.

The Client UI has been fully translated and approved.

File: config/superblocks.ts [hunk 1]
```diff
@@ -169,6 +169,24 @@ export const notAuditedSuperBlocks: NotAuditedSuperBlocks = {
...
+    SuperBlocks.InfoSec,
+    SuperBlocks.MachineLearningPy,
+    SuperBlocks.CollegeAlgebraPy,
+    SuperBlocks.CodingInterviewPrep,
+    SuperBlocks.ProjectEuler,
+    SuperBlocks.TheOdinProject,
+    SuperBlocks.FoundationalCSharp,
+    SuperBlocks.RespWebDesign,
+    SuperBlocks.JsAlgoDataStruct,
+    SuperBlocks.FrontEndDevLibs
```
Comment (naomi-lgbt) [inline config/superblocks.ts:189]:
```suggestion
    SuperBlocks.FrontEndDevLibs,
    SuperBlocks.JsAlgoDataStructNew
```

This allows builds to succeed with `SHOW_UPCOMING_CHANGES`.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Small fix, then this should be good to go.

File: config/superblocks.ts (hunk 1 above)
Comment (larymak) [reply config/superblocks.ts]: Changes have been updated. Kindly confirm if its correctly done

File: config/superblocks.ts [hunk 2]
```diff
@@ -169,6 +169,24 @@ export const notAuditedSuperBlocks: NotAuditedSuperBlocks = {
...
+    SuperBlocks.InfoSec,
+    SuperBlocks.MachineLearningPy,
+    SuperBlocks.CollegeAlgebraPy,
+    SuperBlocks.CodingInterviewPrep,
+    SuperBlocks.ProjectEuler,
+    SuperBlocks.TheOdinProject,
+    SuperBlocks.FoundationalCSharp,
+    SuperBlocks.RespWebDesign,
+    SuperBlocks.FrontEndDevLibs,
+    SuperBlocks.JsAlgoDataStructNew
```
Comment (naomi-lgbt) [inline config/superblocks.ts:189]:
```suggestion
    SuperBlocks.JsAlgoDataStructNew,
    SuperBlocks.JSAlgoDataStruct
```

There are two JS superblocks. Neither of them appear to be translated yet.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: config/superblocks.ts (hunk 2 above)
Comment (larymak) [reply config/superblocks.ts]: Changes updated

**Review by naomi-lgbt — APPROVED** [review]
Looks like it's all good now!

Note that we'll want to translate the meta-tags.json manually, and update the motivation.json with quotes and phrases relevant to the culture.

**Pull request merged**